samples/net/wifi/boards/rd_rw612_bga.conf

```diff
@@ -145,4 +152,4 @@ CONFIG_CODE_DATA_RELOCATION_SRAM=y
 CONFIG_WIFI_NM_WPA_SUPPLICANT_LOG_LEVEL_INF=y
 CONFIG_WIFI_NM_WPA_SUPPLICANT_WQ_PRIO=3
 CONFIG_WIFI_NM_WPA_SUPPLICANT_PRIO=3
-CONFIG_COMMON_LIBC_MALLOC_ARENA_SIZE=60000
+CONFIG_COMMON_LIBC_MALLOC_ARENA_SIZE=80000
```

Comment:
does this include MbedTLS heap too? WPA supplicant itself doesn't need more than 30000.

Reply:
yes, it includes the Mbedtls case, when test the enterprise mode, for example, peap_tls, memory allocation failed may occur.